hack/verify-dirty.sh

```diff
@@ -24,7 +29,7 @@ echo "#############################"
 echo "Verify make configure-bom..."
 echo "#############################"
 make configure-bom
-if ! (git diff --quiet HEAD -- .); then
+if ! (git diff --quiet HEAD -- . "${ignore_file}"); then
```

Comment:
given $ignore_file is literally the only file configure-bom touches, this renders this part of the check useless. But we can improve on this in a followup.

Reply:
Is the `--ignore-matching-lines` option a solution to this? `usebom` will always change this file, so this check will always fail, right?